### PR TITLE
(maint) Fix errant URLS in README / metadata

### PR DESCRIPTION
**Pull request by Iristyle**
- Project page for this module should be the puppetlabs-azure repo
 - README still contained a link to TravisCI at puppetlabs-msazure

### CODE REVIEW

**Comment by gregohardy** [comment]
+1 looks good

File: metadata.json [hunk 1]
```diff
@@ -60,6 +70,6 @@
     }
   ],
   "dependencies": [
-  
+
```
Comment (garethr) [inline metadata.json]: This needs the whitespace to match the checksum for the generated metadata.json from `puppet module build`

**Comment by garethr** [comment]
@Iristyle looks good apart from the comment. If you could include the whitespace. Do you want to include a commit to bump the version to 1.0.1 and I'll get this tagged and released?

File: metadata.json (hunk 1 above)
Comment (Iristyle) [inline metadata.json]: Ah weird, didn't know that ... that's ... peculiar.

Updated per your request and repushed.

**Comment by Iristyle** [comment]
@garethr let's merge this, review and merge #57, then bump to 1.0.1 / tag / release

**Comment by Iristyle** [comment]
Added PR #58 for the `1.0.1` release

File: metadata.json [hunk 2]
```diff
@@ -47,6 +47,16 @@
       "operatingsystemrelease": [
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Windows",
+      "operatingsystemrelease": [
+        "Server 2008 R2",
```
Comment (Iristyle) [inline metadata.json]: `Server 2008` (and older) intentionally omitted here.

**Pull request merged**